feedback/templates/manage/_feedbacks_as_list.html

```diff
@@ -7,6 +7,14 @@
 		feedbacks created with get_feedback_dict
 {% endcomment %}
 
+{% if request.GET.paginate_by %}
+	<script>
+		let selects = document.getElementsByClassName("paginate_by");
```

Comment:
Just to check, are there multiple `class paginate_by` select elements in the page or only one in practice?

Reply:
Multiple, as the pagination element appears both on top and bottom of the page